src/feverous/baseline/drqascripts/build_tfidf.py

```diff
@@ -38,10 +39,13 @@
 
 
 def init(tokenizer_class, db_class, db_opts):
-    global PROCESS_TOK, PROCESS_DB
+    global PROCESS_TOK, PROCESS_DB, DOC2IDX
```

Comment:
that works! Might be worth linking this fix to https://github.com/facebookresearch/DrQA/issues/34, for people facing similar issues when running DrQA on non-Ubuntu machines.